src/main/java/io/lettuce/core/models/events/CommandSucceededEvent.java

```diff
@@ -0,0 +1,31 @@
+package io.lettuce.core.models.events;
+
+import java.time.Duration;
+import java.time.Instant;
+
+/**
+ * @author sokomishalov
+ */
+public class CommandSucceededEvent<K, V, T> extends CommandBaseEvent<K, V, T> {
+
+    private final Instant startedAt;
```

Comment:
Let's keep timestamps as `long` to reduce allocations.

Reply:
fixed